### PR TITLE
ch.mobility.standorte

### PR DESCRIPTION
**Pull request by rebert**
[try](https://mf-geoadmin3.dev.bgdi.ch/src/?api_url=%2F%2Fmf-chsdi3.int.bgdi.ch%2Fltrea_mobility&wms_url=%2F%2Fwms-bgdi.dev.bgdi.ch%2Fltrea_test%3Flang%3Dde&topic=ech&lang=de&bgLayer=ch.swisstopo.pixelkarte-grau&layers=ch.swisstopo.zeitreihen,ch.bfs.gebaeude_wohnungs_register,ch.bav.haltestellen-oev,ch.astra.wanderland,ch.mobility.standorte&layers_visibility=false,false,false,false,true&layers_timestamp=18641231,,,,&X=200423.28&Y=600858.66&zoom=7&catalogNodes=687,720)

### CODE REVIEW

**Comment by AFoletti** [comment]
customer sent some feedbacks. Please have a look at JIRA ticket :smile:

File: chsdi/templates/htmlpopup/mobility_standorte.mako [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+    <% c['stable_id'] = True %>
+    <tr><td class="cell-left">${_('ch.mobility.standorte.name')}</td>           <td>${c['attributes']['name'] or '-'}</td></tr>
+    <tr><td class="cell-left">${_('ch.mobility.standorte.number')}</td>         <td>${c['attributes']['number'] or '-'}</td></tr>
+    <tr><td class="cell-left">${_('ch.mobility.standorte.location')}</td>       <td>${c['attributes']['street'] or '-'}, ${c['attributes']['location'] or '-'}</td></tr>
+    <tr><td class="cell-left">${_('ch.mobility.standorte.lon')}</td>            <td>${c['attributes']['lon'] or '-'}</td></tr>
+    <tr><td class="cell-left">${_('ch.mobility.standorte.lat')}</td>            <td>${c['attributes']['lat'] or '-'}</td></tr>
+    <tr><td class="cell-left">${_('ch.mobility.standorte.link')}</td>           <td><a target="_blank" href="https://www.mobility.ch/${lang}/privatkunden/standorte?station_id=${c['attributes']['number'] or ''}">Link</a></td></tr>
```
Comment (loicgasser) [inline chsdi/templates/htmlpopup/mobility_standorte.mako]: only fr, de, en
please redirect
rm -> de
it -> fr

File: chsdi/models/vector/dritte.py [hunk 2]
```diff
@@ -41,6 +40,40 @@ class FEUERSTELLEN(Base, Vector):
...
+    __template__ = 'templates/htmlpopup/mobility_standorte.mako'
+    __bodId__ = 'ch.mobility.standorte'
+    __label__ = 'name'
+    __returnedGeometry__ = 'the_geom_point'
+    id = Column('bgdi_id', Integer, primary_key=True)
+    name = Column('name', Unicode)
+    number = Column('number', Numeric)
+    location = Column('location', Unicode)
+    lat = Column('lat', Numeric)
+    lon = Column('lon', Numeric)
```
Comment (loicgasser) [inline chsdi/models/vector/dritte.py]: Use Float instead

File: chsdi/models/vector/dritte.py [hunk 3]
```diff
@@ -41,6 +40,40 @@ class FEUERSTELLEN(Base, Vector):
 register('ch.tamedia.schweizerfamilie-feuerstellen', FEUERSTELLEN)
 
 
+class MobilityStandorte:
+    __tablename__ = 'standorte_tooltip'
+    __table_args__ = ({'schema': 'mobility', 'autoload': False, 'extend_existing': True})
```
Comment (loicgasser) [inline chsdi/models/vector/dritte.py]: you don't need extend_existing

Comment (rebert) [inline chsdi/models/vector/dritte.py]: witout extend_existing it does not work

Comment (loicgasser) [inline chsdi/models/vector/dritte.py]: Indeed both models refer to the same table

**Pull request merged**